ix-dev/community/fscrawler/app.yaml

```diff
@@ -0,0 +1,33 @@
+app_version: '2.9'
```

Comment:
App version does not really reflect the tag we are using - is that intended?

Reply:
Fixed!